tests/fpga_system_tests/emu_macro/simctrl.pre.yaml

```diff
@@ -82,6 +89,191 @@ digital_probes:
     width: 10
     signed: 1
 
+  # handling of rare cases in analog_slice
+  t_samp_idx_0:
+    abspath: 'tb_i.top_i.iacore.genblk1[0].analog_slice_i.t_samp_idx'
```

Comment:
these probes could likely be commented out, but it's interesting to see when "unusual" events happen that mess up the sequencing of the ADC samples, since that contributes to BER